metricflow-semantics/metricflow_semantics/query/query_parser.py

```diff
@@ -524,6 +521,15 @@ def _parse_and_validate_query(
             queried_semantic_models=query_resolution.queried_semantic_models,
         )
 
+    def build_query_spec_for_group_by_metric_source_node(
+        self, group_by_metric_spec: GroupByMetricSpec
+    ) -> MetricFlowQuerySpec:
+        """Query spec that can be used to build a source node for this spec in the DFP."""
```

Comment:
micro-nit, I don't like acronyms, it's a personal problem, so if this makes the line too long you can ignore it:
```suggestion
        """Query spec that can be used to build a source node for this spec in the DataflowPlanBuilder."""
```